packages/artifact_proxy/bin/server.dart

```diff
@@ -0,0 +1,55 @@
+// ignore_for_file: avoid_print
+
+import 'dart:io';
+
+import 'package:args/args.dart';
+import 'package:artifact_proxy/artifact_proxy.dart';
+import 'package:shelf/shelf.dart';
+import 'package:shelf/shelf_io.dart' as shelf_io;
+import 'package:shelf_hotreload/shelf_hotreload.dart';
+import 'package:yaml/yaml.dart';
+
+Future<void> main(List<String> args) async {
+  final parser = ArgParser()
+    ..addOption(
+      'config',
+      defaultsTo: 'config.yaml',
+      help: 'Path to config file.',
+    )
+    ..addFlag(
+      'record',
+      help: 'Record requests into config file.',
+    )
+    ..addFlag(
+      'watch',
+      help: 'Whether to watch for changes and hot-reload.',
+    );
+
+  final results = parser.parse(args);
+
+  if (results.rest.isNotEmpty) {
+    print(parser.usage);
+    exit(1);
+  }
+
+  final shouldWatch = results['watch'] as bool;
+  final configPath = results['config'] as String;
+  final config = loadYaml(File(configPath).readAsStringSync()) as Map;
+  final handler = artifactProxyHandler(config: config);
+  final ip = InternetAddress.anyIPv6;
+  final port = int.parse(Platform.environment['PORT'] ?? '8080');
```

Comment:
Why make this env vs the others args?

Reply:
Out of habit but can change to make this a command-line option although I'm not sure there's much value in this being a CLI anymore 🤷

Reply:
Please feel empowered to cut everything. :) We can always add ti back, but when we look at it 3mo from now we won't be able to remove it for fear that something somewehre is using it. 🤣